### ISSUE TITLE
Can't add historical values in a table

### ISSUE THREAD

**Issue by IonuDragomir1** [frangoteam/FUXA#764]
Hello,

I have added on my interface two tables. One displays some real-time values, and the other one should display historical values. 
As you can see in the below image, I managed to add three tags on Data table by clicking on + button from the left  lower corner of Table customizer window. 

![image](https://user-images.githubusercontent.com/58516894/228528562-30a94dc9-bda6-4023-9e57-7b3165f086e7.png)

The + button doesn't exist on History table's Table customizer window. Therefore I can't add any tags to be displayed in History table. 

![image](https://user-images.githubusercontent.com/58516894/228529093-244e2438-0f4a-4a31-a0bc-b47b33b081a1.png)

Am I missing something or this is a bug?

**Comment by unocelli** [comment]
Hi,
for the History table you can only add columns with Tags, the rows are filled with the history values: timestamp, values

**Comment by IonuDragomir1** [comment]
Thanks for replying, @unocelli!
Yes, it was my mistake. I managed to add historical values, but I encountered a problem. When the table control is added on interface, columns order is: timestamp and label. If with this column order, I add a tag for label (Tags) column, everything is OK. This is what I get with default columns order:

![image](https://user-images.githubusercontent.com/58516894/229069369-0b038a29-5ea7-4920-9300-4b51cd5537f7.png)

The problem appears if switch columns order. This is what I get with switched columns order:

![image](https://user-images.githubusercontent.com/58516894/229068749-8763b3f8-bcfe-4606-a8f4-e030115e3282.png)

I changed column order by deleting the Data/Time column and then added it back.
